examples/README.md

```diff
@@ -1,8 +1,37 @@
 ## Run Spline examples 
-To run Spline example, download the Spline source code from GitHub and switch to the `examples` directory.     
-```shell script
-cd $SPLINE_PROJECT/examples
-```
+
+1. Make sure the Spline Producer instance is running ([see instructions](https://absaoss.github.io/spline/0.4.html#start-spline-server))
+
+2. Download the Spline source code from GitHub and switch to the `examples` directory     
+    ```shell script
+    git clone git@github.com:AbsaOSS/spline.git
+    cd spline-spark-agent
+    git checkout release/0.4.2
```

Comment:
Why not `git checkout release/0.4` to include any hotfixes?

Reply:
Just to be precise. If you clone `release/0.4` the artifacts will of a snapshot version. Although there is no technical difference in comparison to the latest tag, but it might lead to confusion for some users and make troubleshooting more complicated.

Reply:
ok, feel free to merge